account_due_list_days_overdue/tests/test_account_due_list_days_overdue.py

```diff
@@ -51,8 +50,8 @@ def setUp(self):
             'account_id': self.sales_account.id,
             'invoice_id': self.invoice.id,
             'name': 'product that cost 100'})
-        workflow.trg_validate(self.uid, 'account.invoice', self.invoice.id,
-                              'invoice_open', self.cr)
+        self.invoice.action_invoice_open()
+        self.assertEqual(self.invoice.state, 'open')
```

Comment:
This is the setup method, I would move or remove this check.

Reply:
Done